modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/SpringCodegen.java

```diff
@@ -115,7 +115,7 @@ public SpringCodegen() {
         cliOptions.add(CliOption.newBoolean(USE_TAGS, "use tags for creating interface and controller classnames", useTags));
         cliOptions.add(CliOption.newBoolean(USE_BEANVALIDATION, "Use BeanValidation API annotations", useBeanValidation));
         cliOptions.add(CliOption.newBoolean(PERFORM_BEANVALIDATION, "Use Bean Validation Impl. to perform BeanValidation", performBeanValidation));
-        cliOptions.add(CliOption.newBoolean(IMPLICIT_HEADERS, "Use of @ApiImplicitParams for headers.", implicitHeaders));
+        cliOptions.add(CliOption.newBoolean(IMPLICIT_HEADERS, "Skips header parameters in the generated api methods.", implicitHeaders));
```

Comment:
I like the revised description but what about still mentioning `@ApiImplicitParams`. Here is what I suggested:

```
Skip header parameters in the generated API methods using @ApiImplicitParams annotation
```

Reply:
+1 makes sense. Ill update the commit.